libfreerdp/core/nla.c

```diff
@@ -97,9 +94,7 @@
  *
  */
 
-#define NLA_PKG_NAME NEGO_SSP_NAME
-
-#define TERMSRV_SPN_PREFIX "TERMSRV/"
+#define NLA_PKG_NAME AUTH_PKG_SPNEGO
```

Comment:
is it me or it SPNego the default for NLA ?

Reply:
Wasn't it always the default?